tests/lightning_qubit/test_measurements_class.py

```diff
@@ -19,20 +19,28 @@
 import numpy as np
 import pennylane as qml
 import pytest
-from conftest import LightningDevice  # tested device
+from conftest import LightningDevice, device_name  # tested device
 from pennylane.devices import DefaultQubit
 from pennylane.measurements import VarianceMP
 from scipy.sparse import csr_matrix, random_array
 
-from pennylane_lightning.lightning_qubit import LightningQubit
+try:
+    from pennylane_lightning.lightning_qubit_ops import (
+        MeasurementsC64,
+        MeasurementsC128,
+    )
+except ImportError:
+    pass
+
+from pennylane_lightning.lightning_qubit2 import LightningQubit2
 from pennylane_lightning.lightning_qubit._measurements import LightningMeasurements
 from pennylane_lightning.lightning_qubit._state_vector import LightningStateVector
 
-if not LightningQubit._CPP_BINARY_AVAILABLE:
-    pytest.skip("No binary module found. Skipping.", allow_module_level=True)
+if LightningDevice != LightningQubit2:
+    pytest.skip("Exclusive tests for lightning.qubit2. Skipping.", allow_module_level=True)
 
-if LightningDevice != LightningQubit:
-    pytest.skip("Exclusive tests for lightning.qubit. Skipping.", allow_module_level=True)
+if not LightningDevice._CPP_BINARY_AVAILABLE:
+    pytest.skip("No binary module found. Skipping.", allow_module_level=True)
```

Comment:
Same here. We could have let it be LightningQubit.  Only the binaries are relevant for these unit tests.